src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js

```diff
@@ -31,11 +31,35 @@ class IbexaCustomTagEditing extends Plugin {
         conversion.for('editingDowncast').elementToElement({
             model: 'customTag',
             view: (modelElement, { writer: downcastWriter }) => {
+                const customTagName = modelElement.getAttribute('customTagName');
                 const container = downcastWriter.createContainerElement('div', {
                     'data-ezelement': 'eztemplate',
-                    'data-ezname': modelElement.getAttribute('customTagName'),
-                    class: 'ez-custom-tag',
+                    'data-ezname': customTagName,
+                    class: 'ibexa-custom-tag',
                 });
+                const header = downcastWriter.createUIElement('div', { class: 'ibexa-custom-tag__header' }, function(domDocument) {
+                    const domElement = this.toDomElement(domDocument);
+
+                    domElement.innerHTML = `
```

Comment:
Cannot you put it in twig and pass as `data-template`?

Reply:
Those should be something outside of the twig templates.